### PR TITLE
Add tblite project to registry

### PR DESCRIPTION
**Pull request by awvwgk**
This is my largest fpm project so far with more than 500 source files in 6 fpm projects. Still a bit experimental, but is already integrated in projects like [DFTB+](https://github.com/dftbplus/dftbplus) and [QCxMS](https://github.com/qcxms/qcxms). I can say that this project would not have been possible without fpm.

- source at [awvwgk/tblite](https://github.com/awvwgk/tblite)
- supports fpm, meson and CMake as build system
- GCC 8 and newer and Intel 18 and newer are supported
- documentation at https://tblite.readthedocs.io
- implements [10.1021/acs.jctc.8b01176](https://pubs.acs.org/doi/10.1021/acs.jctc.8b01176) (open access)

### CODE REVIEW

**Comment by ivan-pi** [comment]
Out of curiosity, can you summarize the main advantages of developing using fpm?

**Review by certik — APPROVED** [review]

**Comment by awvwgk** [comment]
> Out of curiosity, can you summarize the main advantages of developing using fpm?

Fpm has been incredible helpful for bootstrapping projects, my usual strategy with fpm now is
1. create minimal package manifest (*name*, *build.link*, *dependencies*, *dev-dependencies*, nothing more)
2. add `src` and `test` folder, I'm using [mctc-lib](https://github.com/grimme-lab/mctc-lib) to setup my testing framework, but it works similar to [test-drive](https://github.com/awvwgk/test-drive)
3. populate unit tests in `test` and write corresponding implementation in `src` (repeat)
4. create a command line driver in `app`

Up to this point there is no need for writing any build files or CI setup and integration, thrashing the project at this stages cost no effort, because I have not really invested anything yet in the project except for coding (and this is the easily reusable part). Being able to start quickly and also to stop again without losing much time enables me to do a lot of exploratory work.

After this point, I usually ditch fpm for a while and write a complete set of meson build files, especially for C and Python integration and to actually compile the project in production settings. But fpm always comes in handy if I just want to quickly test something, like checking out a new idea or algorithm.

**Review by LKedward — APPROVED** [review]

**Review by milancurcic — APPROVED** [review]
Great addition, thanks!
